test/gtest/ucp/ucp_test.cc

```diff
@@ -217,15 +221,13 @@ ucs_status_t ucp_test::request_process(void *req, int worker_index, bool wait)
             progress(worker_index);
             status = ucp_request_check_status(req);
         } while ((status == UCS_INPROGRESS) && (ucs_get_time() < deadline));
+        EXPECT_NE(UCS_INPROGRESS, status) << "request " << req
+                                          << " did not complete on time";
```

Comment:
let's unify with `test/gtest/ucp/test_ucp_sockaddr.cc:617`, i.e. (not)completed by some operations:
`wasn't completed on time`